src/ext/github/client.js

```diff
@@ -5,8 +5,9 @@ require('superagent-bluebird-promise');
 
 var config = require('./../../../config');
 
-function buildApiRequest(url) {
-    var req = request.get(url);
+function buildApiRequest(path) {
+    var url = config.api.github.baseUrl || 'https://api.github.com';
```

Comment:
@jasonbellamy sorry for my code snippet proposal, I think the `url` var should stay outside the `buildApiRequest` function